config/routes.rb

```diff
@@ -376,5 +376,8 @@
 
   get "/how-to-set-up-your-programme", to: "step_by_step#show", as: :step_by_step
 
+  get "/assets/govuk/assets/fonts/:name.:extension", to: redirect("/api-reference/assets/govuk/assets/fonts/%{name}.%{extension}")
```

Comment:
not sure we want this? every asset will require an extra request assuming it's not cached. I feel like this is a workaround and doesn't solve the actual problem we have introduced.

Reply:
The problem is that the gem `govuk_tech_docs` hard-codes its assets to be in `/assets/govuk/assets/`, and I am not sure it's a good place for something that's a small part of our application.

`govuk_tech_docs` in general doesn't seem to believe it should be just a small part of an application. But I think it is a pretty good idea to have the tech docs be part of what we release, and this seemed like the simplest solution. I guess we could make it a separate mini-service in paas, but that would require a lot of dev ops fun :/

Reply:
Edit: We could get rid of it when https://github.com/alphagov/tech-docs-gem/pull/197 goes in, or something similar to it.